rotkehlchen/interfaces.py

```diff
@@ -0,0 +1,44 @@
+import abc
+from typing import Optional
+
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.types import Price, Timestamp
+
+
+class PriceOracleInterface(metaclass=abc.ABCMeta):
```

Comment:
Needs docstring. Is this for Historical? Then add it to the name. I am not sure when looking at it if it's for historical or current.